### ISSUE TITLE
Add an optional value to -e

### ISSUE THREAD

**Issue by sfussenegger** [keenerd/jshon#36]
This is the JSON snippet I'm working on (as returned by aws cli):

```
{
  "Outputs": [
   {
    "OutputValue": "value1",
    "OutputKey": "key1"
   },
   {
    "OutputValue": "value2",
    "OutputKey": "key2"
   }
  ]
}
```

I want jshon to get the OutputValue where the OutputKey is 'key2'. 'value2' that is. Currently, I don't see a way to do this, so I'd like to suggest an optional value to the -e parameter:

`jshon -e Outputs -a -e OutputKey=key2 -p -e OutputValue -u`

**Comment by Napsty** [comment]
Good idea. I'd love to see this select (kind of conditional) parameter `-w` in the future!
